Makefile

```
@@ -112,8 +112,8 @@ $(WORKDIR)/ipk/content/app/cobalt/content/web/adblock: $(BUILD_WEBOS_YOUTUBE_APP
 	fi
 
 	rm -f $(WORKDIR)/ipk/drm.nfz
-	sed -i 's/YouTube/YouTube Cobalt AdBlock/g' $(WORKDIR)/ipk/appinfo.json
-	jq 'del(.fileSystemType)' < $(WORKDIR)/ipk/appinfo.json > $(WORKDIR)/ipk/appinfo2.json
+	sed -i 's/"title":"YouTube"/"title":"YouTube Cobalt AdBlock"/g' $(WORKDIR)/ipk/appinfo.json
```

Hi,
Thanks for the contribution
Is it possible to have instead:
```suggestion
	sed -i 's/"YouTube"/"YouTube Cobalt AdBlock"/g' $(WORKDIR)/ipk/appinfo.json
```

In my case, I have this line too:
```
"dialAppName": "YouTube",
```
which I would like to rename to `"dialAppName": "YouTube Cobalt AdBlock",`

---

Why do you want to change `dialAppName`?